addons/l10n_id/models/account_move.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import base64
+from odoo import models
+from odoo.exceptions import UserError
+from odoo.tools.image import image_data_uri
+
+QRIS_API_URL = "https://qris.online/restapi/qris/show_qris.php"
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _generate_qr_code(self, silent_errors=False):
+        """Override
+
+        Using QRIS API in order to generate the QR code
+        """
+        self.ensure_one()
+        if self.company_id.country_code == "ID":
```

Comment:
I would also add a check on if `self.display_qr_code` is false.
Just to be safe, in the same way it is done in the original method